### PR TITLE
CE-647 Replacing create-canvas-and-add-note with create-canvas-and-add-text …

### PR DESCRIPTION
**Pull request by Codiegladney-bluescape**
…since Note was removed in March release

### CODE REVIEW

File: site-templates/pages/create-canvas-and-add-text.gomd [hunk 1]
```diff
@@ -346,19 +356,26 @@ From this response, you can obtain the newly created Canvas ID: ['data']['id']
...
+                "a": 1
+            },
             "backgroundColor": {
-                "r": 56,
-                "g": 138,
-                "b": 135,
+                "r": 0,
+                "g": 0,
+                "b": 0,
                 "a": 1
```
Comment (vlad-bluescape) [inline site-templates/pages/create-canvas-and-add-text.gomd:379]: @Codiegladney-bluescape : Change this value to "0". Alpha channel "0" means transparent. Currently the transparent background does not work for Text (shows black), but let's set it correctly for when the bug is fixed.

Comment (vlad-bluescape) [reply site-templates/pages/create-canvas-and-add-text.gomd:379]: Sorry, this is the response....

**Review by vlad-bluescape — APPROVED** [review]
looks good. Merge into v3beta, please

**Review by Jackychen-bluescape — APPROVED** [review]
LGTM

**Pull request merged**